components/table/table/tableFilters/TableFilters.module.scss

```diff
@@ -1,5 +1,85 @@
 @import '../..../../../../../styles/';
 
+.v2 {
+	@include flex(row, space-between, center);
+	padding: 1.25rem;
+
+	.details {
+		@include flex(column, center, flex-start);
+		gap: 0.5rem;
+		width: 30%;
+	}
+	.filters {
+		@include flex(row, flex-end, center);
+		gap: 0.5rem;
+		width: 30%;
+
+		button.icon-button {
+			background: transparent;
+			border: 1px solid $mono-color2;
+			height: 1.875rem;
+			& > [data-elem='component2'] {
+				& > [data-elem='title'] {
+					color: $text-color;
+					font-size: 0.875rem;
+					font-weight: 400;
+				}
+			}
+		}
+		.icon {
+			width: 1rem;
+			height: 1rem;
+			flex-shrink: 0;
+		}
+
+		.column-dropdown {
+			@include flex(row, center, center);
+			> [data-elem='header'] {
+				> [data-elem='select'] {
+					background-color: transparent;
+					border: none;
+					color: $text-color;
+					padding: 0rem;
+					width: auto;
+				}
+			}
+
+			.item {
+				height: 3rem;
+				> [data-elem='title'] {
+					@include ellipsis;
+				}
+			}
+		}
+	}
+	.search {
+		width: 30%;
```

Comment:
Same flexible conditions here. In case of doubts in the flex ratio, discuss with designer on the proportion given to each section. So can be adjusted accordingly.